microsoft-edge/devtools-guide-chromium/rendering-tools/index.md

```diff
@@ -43,7 +43,7 @@ Take a recording in the **Performance** tool and look for suspiciously long `Eva
 <!--todo: add Recording section when available  -->  
 <!--todo: add Profile JavaScript (JS Profiler) section when available  -->  
 
-f you noticing quite a bit of jank in your JavaScript, you may need to take your analysis to the next level and collect a JavaScript CPU profile.  CPU profiles show where runtime is spent within the functions of your page.  Learn how to create CPU profiles in [Speed Up JavaScript Runtime][DevtoolsRenderingToolsJavascriptRuntime].
+If you notice quite a bit of jank in your JavaScript, you may need to take your analysis to the next level and collect a JavaScript CPU profile.  CPU profiles show where runtime is spent within the functions of your page.  Learn how to create CPU profiles in [Speed up JavaScript runtime][DevtoolsRenderingToolsJavascriptRuntime].
```

Comment:
I will (or you could) full-text search "jank" in repo, two articles use the word in visible text, needs to be defined in both articles.  wiktionary: "jank ( uncountable ) ( computing, slang, rare) Perceptible pause in the smooth rendering of a software application 's user interface due to slow operations or poor interface design. quotations ".  I plan to enter Suggestions in two articles in this PR.

Reply:
```suggestion
If you notice quite a bit of jank (interruptions of rendering) in your JavaScript, you may need to take your analysis to the next level and collect a JavaScript CPU profile.  CPU profiles show where runtime is spent within the functions of your page.  Learn how to create CPU profiles in [Speed up JavaScript runtime][DevtoolsRenderingToolsJavascriptRuntime].
```